org.adempiere.base/src/org/compiere/util/Ini.java

```diff
@@ -794,4 +798,76 @@ public static String getPropertyFileName()
 	{
 		return s_propertyFileName;
 	}
+
+	public static String getVar(String secretVar) {
+		String cmd = getUtilsCmd("getVar");
+		String[] command = new String[] {
+				cmd,
+				secretVar
+		};
+		String retValue = runCommand(command);
+		return retValue;
+	}
+
+	public static void setVar(String secretVar, String secretValue) {
+		String cmd = getUtilsCmd("setVar");
+		String[] command = new String[] {
+				cmd,
+				secretVar,
+				secretValue
+		};
+		@SuppressWarnings("unused") // used for debugging purposes
+		String retValue = runCommand(command);

```

Comment:
@CarlosRuiz-globalqss don't need this one. eclipse already support
in case we use a IDE hasn't this feature then debug at end of runCommand better

![image](https://user-images.githubusercontent.com/4666285/103646657-9a705180-4f8c-11eb-89be-4ee525668930.png)